content/cpp/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'round()'
+Description: 'Returns the integer that is nearest to the argument, with halfway cases rounded away from the ending zero.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`round()`** function returns the integer that is closest to the argument, with halfway cases rounded away from the ending zero.
+
+## Syntax
+
+```pseudo
+round(num);
+```
+
+The `num` parameter must be a `double`,`float`, or `long double`. The return value will be an integer. If `num` is a halfway case where the decimal place is `.5` or higher, then the closest integer above `num` is returned.
```

Comment:
~If `num` is a halfway case where the decimal place is `.5` or higher, then the closest integer above `num` is returned.~

Consider this instead:

"If the decimal in `num` is 0.5 or higher, the closest integer greater than `num` is returned."